pages/common/meson.md

```diff
@@ -7,7 +7,7 @@
 
 `meson init --language={{c}} --name={{myproject}} --version={{0.1}}`
 
-- Configure build directory with default values:
+- Configure the builddir with default values:
```

Comment:
This doesn't make sense either. What about this?
```suggestion
- Configure the build directory with default values:
```

...unless `builddir` is a specific technical term in the contact of `meson`?

Reply:
I made this exact change, but @nicokosi pointed at that, in fact, it _is_ specific to meson: https://github.com/tldr-pages/tldr/pull/5848#discussion_r624564008